configsuite/validator.py

```diff
@@ -75,6 +75,9 @@ def _validate(self, config, schema):
         if self._apply_validators and valid:
             valid &= self._context_validation(config, schema)
 
+        if self._apply_validators and valid:
+            valid &= self._length_validation(config, schema)
```

Comment:
It's too late now, but should we not stick to logical operators for boolean values?

`valid = valid and self._length_validation(config, schema)`

Reply:
I'll stick with the above in this PR due to my earlier doings right above. However, I'll definitively accept any patch removing the bitwise operators. Or, if you expand your argument of why we should do so, I might do it myself. But either way, I think it belongs in a separate PR 👍